### PR TITLE
feat: Front profil details

### PR DESCRIPTION
**Pull request by MTthoas**
image

### CODE REVIEW

File: next-app/src/components/header/Header.tsx [hunk 1]
```diff
@@ -26,6 +28,8 @@ const HeaderLinks: NavLink[] = [
 ];
 
 export default function Header(): JSX.Element {
+  const { address } = useAccount();
+  console.log(address);
```
Comment (Ronfflex) [inline next-app/src/components/header/Header.tsx:32]: Console log à retirer

**Review by Ronfflex — CHANGES_REQUESTED** [review]
console.log to remove

File: next-app/src/components/header/Header.tsx (hunk 1 above)
Comment (MTthoas) [inline next-app/src/components/header/Header.tsx:32]:
```suggestion
```

File: next-app/src/app/profil/page.tsx [hunk 2]
```diff
@@ -16,16 +19,13 @@ export default function Profil() {
...
     queryKey: ["getUserByAddress", address],
     queryFn: () => getUserByAdress(address!),
     enabled: !!address,
   });
 
-  const { data: users } = useQuery({
-    queryKey: ["users"],
-    queryFn: getUsers,
-  });
+  console.log(user);
```
Comment (Guiiz94) [inline next-app/src/app/profil/page.tsx:28]: console.log to remove here too

Comment (MTthoas) [inline next-app/src/app/profil/page.tsx:28]:
```suggestion
```

File: next-app/src/components/header/Header.tsx (hunk 1 above)
Comment (Guiiz94) [inline next-app/src/components/header/Header.tsx]:
```suggestion
```

**Pull request merged**